project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1320 @@
+        apply_logic: 
+          - HVAC Has Shared System|None
+          - HVAC Has Ducts|No
+          - not: HVAC Heating Type And Fuel|Electricity MSHP
+        costs: *costs_mshp_ductless_8-33_HSPF
+        lifetime: *lifetime_HP
+
+    #### 1.4 High-Efficiency Heat Pump With Electric Backup, 2 options  ####
+      - &1_high_eff_ashp_ducted
+        option: HVAC Heating Efficiency|ASHP, SEER 20, 11 HSPF, CCHP, Max Load, Supplemental Backup Sizing
+        apply_logic:
+          - HVAC Has Ducts|Yes
+          - not: Heating Fuel|None
+          - not: HVAC Heating Type And Fuel|Electricity ASHP
+        costs: &costs_ashp_ducted_SEER_20_11_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. January 2019 $ inflated to April 2023 $ using 1.21 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 10229.58 # for HSPF 11
+            multiplier: Fixed (1)
+          - value: 167.64
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: *lifetime_HP
+
+      - &2_high_eff_mshp_ductless
+        option: HVAC Heating Efficiency|MSHP, SEER 20, 11 HSPF, CCHP, Max Load, Supplemental Backup Sizing
+        apply_logic:
+          - HVAC Has Ducts|No
+          - not: Heating Fuel|None
+          - not: HVAC Heating Type And Fuel|Electricity MSHP
+        costs: &costs_mshp_ductless_SEER_20_11_HSPF
+        # Source: Custom regression by Brennan Less. Ductless heat pump project costs were regressed on nameplate tons,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. January 2019 $ inflated to April 2023 $ using 1.21 factor.
+        # Ductless MSHP n=173, HSPF median 11, range 9.3–14.2. Implicitly includes multiple zones and electrical upgrades.
+        # Regression results in costs that are 15–30% higher than costs from The Heat Pump Store as documented in https://redwoodenergy.net/wp-content/uploads/2021/02/Pocket-Guide-to-All-Electric-Retrofits-of-Single-Family-Homes.pdf 
+        # LBNL data were not sufficient to include a relationship between SEER or HSPF and cost, so cost deltas for higher 
+        # and lower HSPF were added using the relationship between HSPF and cost for ductless MSHPs in REMDB: 2.2% addition or subtraction per point
+        # according to how many points above or below the HSPF rating is from 10.5 (https://remdb.nrel.gov/).
+          - value: 2844.81
+            multiplier: Fixed (1)
+          - value: 369.43
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: *lifetime_HP
+
+    #### 2.1 Electric Water Heating, 4 options  ####
+      - &1_water_heater_fuel 
+        option: Water Heater Fuel|Electricity
```

Comment:
```
apply_logic:
  - not: Water Heater Fuel|Electricity
```

Reply:
Actually, remove this option. It doesn't do anything

Reply:
removed &1_water_heater_fuel